src/Core/Services/Implementations/AzureQueueMailService.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Storage.Queues;
+using Bit.Core.Models.Mail;
+using Bit.Core.Settings;
+using Newtonsoft.Json;
+
+namespace Bit.Core.Services
+{
+    public class AzureQueueMailService : AzureQueueService<IMailQueueMessage>, IMailEnqueuingService
+    {
+        protected override QueueClient QueueClient { get; }
+        protected override JsonSerializerSettings JsonSettings { get; } = new JsonSerializerSettings
```

Comment:
You may not actually want this, using the arrow function in the short form of doing this: 
```csharp
protected override JsonSerializerSettings JsonSettings
{
    get 
    { 
        return new JsonSerializerSettings();
    }
}
```

This would mean you would be getting a new instance everytime it is requested. In the original it would just be created once at the instantiation of `AzureQueueMailService` and then the same instance would be given back each time.

Reply:
True, ignore my comment.

Reply:
Your constructor suggestion solves this